### ISSUE TITLE
Facetime HD cameras of new M1 macs always detected as on

### ISSUE THREAD

**Issue by chrsvgt** [sindresorhus/is-camera-on#4]
The new M1 MacBook Pro camera hardware is always detected as "on". 

So far a quite extensive research on what has changed has been unsuccessful. Perhaps you may have any idea what needs to be changed in this great tool?

**Comment by sindresorhus** [comment]
I don't have an M1 Mac, so I, unfortunately, cannot look into this issue.

**Comment by davestrand** [comment]
Yeah, I am having the same issue with the M1 always reporting 1 on the on.pointee check.  But check out this discussion..  https://github.com/wouterdebie/onair/issues/3

I have attempted to hack away at your code to make it work but I can't seem to differentiate when the facetime camera is on or off on a M1 to make this code work.
